arm/cdn/customdomains.go

```diff
@@ -118,14 +122,18 @@ func (client CustomDomainsClient) CreateResponder(resp *http.Response) (result a
 	return
 }
 
-// DeleteIfExists sends the delete if exists request.
+// DeleteIfExists sends the delete if exists request.This method handles
```

Comment:
Yes, Polling will still be there for lro, it can't be cancelled. It should not be "optional" in comment. I have updated it to "can ignore polling". Is this making sense? 

```
+// DeleteIfExists sends the delete if exists request.This method handles
+// polling itself for long-running operations. User can ignore polling for
 +// long-running calls by passing cancel channel in last argument of this
 +// method (cancel <-chan struct{}). Pass cancel channel if don't want to wait
 +// for polling to complete, else pass nil.
```